core/domain/user_services.py

```diff
@@ -2344,14 +2344,15 @@ def allow_user_to_review_translation_in_language(
     user_contribution_rights = get_user_contribution_rights(user_id)
     allowed_language_codes = set(
         user_contribution_rights.can_review_translation_for_language_codes)
-    allowed_language_codes.add(language_code)
+    if language_code is not None:
+        allowed_language_codes.add(language_code)
     user_contribution_rights.can_review_translation_for_language_codes = (
         sorted(list(allowed_language_codes)))
     _save_user_contribution_rights(user_contribution_rights)
 
 
 def remove_translation_review_rights_in_language(
-    user_id: str, language_code_to_remove: str
+    user_id: str, language_code_to_remove: Optional[str]
```

Comment:
This is the same case as mentioned for `allow_user_to_review_translation_in_language` above

Reply:
Yup, it shouldn't be None. That would be an error case. Worth double-checking the callsites to ensure that that's not a possible argument.

Reply:
Raised an exception PTAL!